### PR TITLE
Cli more flags

### PR DESCRIPTION
**Pull request by amn41**
**Proposed changes**:
- user can choose if they want to finetune the model after giving feedback (deafult False)
- user can provide a model to load instead of pre-training from data. This should probably be made a mutually exclusive group with all the training-related cmdline args
- add exit code flag to evaluate.py

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: rasa_core/evaluate.py [hunk 1]
```diff
@@ -60,7 +60,11 @@ def create_argument_parser():
             type=str,
             default="failed_stories.txt",
             help="output path for the failed stories")
-
+    parser.add_argument(
```
Comment (tmbo) [inline rasa_core/evaluate.py]: this has already been added in #966 as `--fail_on_prediction_errors `

File: rasa_core/train.py [hunk 2]
```diff
@@ -82,6 +82,15 @@ def create_argument_parser():
             default=False,
             action='store_true',
             help="enable online training")
+    parser.add_argument(
```
Comment (tmbo) [inline rasa_core/train.py]: should we use the same name as we use in `run`?  `--core`

File: rasa_core/training/online.py [hunk 3]
```diff
@@ -469,8 +469,8 @@ def _write_stories_to_file(export_file_path, sender_id, endpoint):
 
 
 def _predict_till_next_listen(endpoint,  # type: EndpointConfig
-                              sender_id  # type: Text
-                              ):
+                              sender_id,  # type: Text
+                              finetune): # type: Boolean
```
Comment (tmbo) [inline rasa_core/training/online.py]: type needs to come directly after the argument (`):`) needs to go onto the next line

File: rasa_core/evaluate.py (hunk 1 above)
Comment (amn41) [reply rasa_core/evaluate.py]: 👌

**Comment by amn41** [comment]
thanks @tmbo for comments. I just log now that all training params are ignored if loading a pre-trained model

File: rasa_core/train.py [hunk 4]
```diff
@@ -30,7 +30,7 @@ def create_argument_parser():
 
     # either the user can pass in a story file, or the data will get
     # downloaded from a url
-    group = parser.add_mutually_exclusive_group(required=True)
+    group = parser.add_mutually_exclusive_group(required=False)
```
Comment (tmbo) [inline rasa_core/train.py]: how about keeping the exclusive group with required and adding the `--core` to that group - so either you need to supply training data or a model

**Comment by amn41** [comment]
cool @tmbo please have another look

**Pull request merged**